http/_negotiation/encoding.ts

```diff
@@ -158,5 +158,5 @@ export function preferredEncodings(
   return priorities
     .filter(isQuality)
     .sort(compareSpecs)
-    .map((priority) => provided[priorities.indexOf(priority)]);
+    .map((priority) => provided[priorities.indexOf(priority)]!);
 }
```

Comment:
Can you please justify this non-null assertion?

Reply:
`priorities` is generated from the values of `provided` so `provided[priorities.indexOf(priority)]` is always non-null